src/Form/Input.js

```diff
@@ -271,12 +306,14 @@ class Input extends Component {
         <InputContainer styles={rest.styles}>
           {floating && Label}
 
-          {multiline ? <StyledTextArea {...inputProps} /> : <StyledInput {...inputProps} />}
+          {icon && <StyledIcon size={iconSize} color={iconColor} name={icon} />}
+
+          {multiline ? <StyledTextArea {...inputProps} /> : <StyledInput hasIcon={icon} {...inputProps} />}
```

Comment:
Should we pass in a boolean here?
```suggestion
          {multiline ? <StyledTextArea {...inputProps} /> : <StyledInput hasIcon={!!icon} {...inputProps} />}
```

Reply:
Ah yes. That makes more sense.